app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -86,10 +83,21 @@ function fetchNotificationsCount() {
   }
 }
 
-function onClickReaction(event) {
+// map over the array of elements and apply a callback function to click event
+function map(className, callBack) {
```

Comment:
what about `addOnClickHandlerToButtons` ? `map` is a really generic name

Reply:
Yeah, that's a good name.